mmdet/utils/ascend_util.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+
+def set_index(ori_tensor, mask, new_value, neg=False):
```

Comment:
set_index is a bit vague, it seems that this function essentially masks some tensor by mask and add values at that place, similar to torch.where, can we directly use the API of torch.where?
If not, please add docstring and use names like masked_fill or other clearer names.

Reply:
commit